mock-identity-system/src/main/java/io/mosip/mock/identity/system/controller/MockIdentityController.java

```diff
@@ -0,0 +1,52 @@
+package io.mosip.mock.identity.system.controller;
+
+import javax.validation.Valid;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import io.mosip.mock.identity.system.dto.MockAuthData;
+import io.mosip.mock.identity.system.dto.MockAuthDataResponse;
+import io.mosip.mock.identity.system.dto.RequestWrapper;
+import io.mosip.mock.identity.system.dto.ResponseWrapper;
+import io.mosip.mock.identity.system.exception.MockIdentityException;
+import io.mosip.mock.identity.system.service.MockIdentityService;
+import io.mosip.mock.identity.system.util.MockIdentityUtil;
+
+
+@RestController
+@RequestMapping("/")
+public class MockIdentityController {
+
+	@Autowired
+	private MockIdentityService mockIdentityService;
+
+	@PostMapping(value = "mock-identity", consumes = { MediaType.APPLICATION_JSON_VALUE }, produces = {
```

Comment:
Lets rename this to just identity in the URL.
Lets also change the controller and service names to IdentityController, IdentityService, IdentityRepository etc..